web-bundle/src/main/java/com/graphhopper/customspeeds/CustomSpeedsVehicle.java

```diff
@@ -1,17 +1,33 @@
 package com.graphhopper.customspeeds;
 
 import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
 import com.graphhopper.RouterConstants;
+import com.graphhopper.http.TruckAverageSpeedParser;
+import com.graphhopper.routing.util.parsers.CarAverageSpeedParser;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import java.util.EnumSet;
-import java.util.Objects;
+import java.util.*;
 
 public class CustomSpeedsVehicle {
+    private static final Logger logger = LoggerFactory.getLogger(CustomSpeedsVehicle.class);
+
     // TODO support custom speeds for bikes and pedestrians (RAD-6445, RAD-6446)
     public enum VehicleType {
-        CAR,
-        TRUCK,
-        SMALL_TRUCK,
+        CAR(CarAverageSpeedParser.CAR_MAX_SPEED),
+        TRUCK(TruckAverageSpeedParser.EE_TRUCK_MAX_SPEED),
+        SMALL_TRUCK(TruckAverageSpeedParser.EE_TRUCK_MAX_SPEED);
```

Comment:
fyi, i moved this file and `CustomSpeedUtils` from `replica-common` to `web-bundle` so this class could access `TruckAverageSpeedParser` and these max speeds. considered moving the speed constants to `replica-common` instead, but `CustomSpeedsUtils` and `CustomSpeedsVehicle` were already only used in `web-bundle` so this felt a little cleaner